pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -204,18 +204,23 @@ def on_load_checkpoint(self, callback_state: Dict[str, Any]):
         self.best_model_score = callback_state["best_model_score"]
         self.best_model_path = callback_state["best_model_path"]
 
-    def save_checkpoint(self, trainer, pl_module):
+    def save_checkpoint(self, trainer, unused: Optional = None):
```

Comment:
How about keeping the variable name because the name will be still rendered in the docs until it finally gets removed in v1.5?
```suggestion
    def save_checkpoint(self, trainer, pl_module: Optional = None):
```

Reply:
We don't have `Args:` for this function so I don't want users to think the parameter is necessary